bots/tests-invoke

```diff
@@ -249,7 +249,7 @@ class PullTask(object):
                 self.github_status_data["state"] = "success"
         if isinstance(ret, str):
             message = ret
-            mark_failed()
+            ret = 0
```

Comment:
The `ret = 0` seems correct, but why drop the `mark_failed()`? That sounds like it woudl not update the GitHub status accordingly, and thus the status would be "pending" forever?